README.md

```diff
@@ -4,3 +4,22 @@ AzureIntegration
 Features that integrate ASP.NET Core with Azure.
 
 This project is part of ASP.NET Core. You can find samples, documentation and getting started instructions for ASP.NET Core at the [Home](https://github.com/aspnet/home) repo.
+
+
+SiteExtensions
+===
+
+To install ASP.NET Core runtime site extension:
```

Comment:
Add a short description of this site extension. What is it good for and why should I install it (for example)?

Reply:
There would be a blog post about this when preview1 ships. This instruction is for people who know what they need.

Reply:
"To install a nightly preview of the site extension for testing purposes...."

Don't want customers thinking these are the official Preview1 instructions.

Reply:
@Tratcher would only be "nightly" if we were using the https://dotnet.myget.org/F/aspnetcore-dev/ feed. Need to say "prerelease" or some such.